robottelo/hosts.py

```diff
@@ -1776,6 +1787,50 @@ def _swap_nailgun(self, new_version):
         to_clear = [k for k in sys.modules if 'nailgun' in k]
         [sys.modules.pop(k) for k in to_clear]
 
+    def enable_ipv6_http_proxy(self):
+        # Execute procedures for enabling IPv6 HTTP Proxy
+        if all[settings.server.is_ipv6, settings.server.http_proxy_ipv6_url]:
```

Comment:
@ogajduse Yeah, you are right, its not yet because we are still waiting for IPv6 playground in CI to run test execution for IPV6.

I opened this PR for review for these static checks, so that we curb the issues earlier before we run it actually in CI.

Reply:
BTW, the all suggestion is modified now to:
```
all([settings.server.is_ipv6, settings.server.http_proxy_ipv6_url])
```